mautic/mautic
# Fix report points change log graphs

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/6476
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Points log report work with graphs with sum in mysql select. If you 're using data table with graphs, and contact has more companies, then to SQL query are joined two categories tables.  Then points sum  number are not correct. This PR removes categories tables from graphs queries.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1.  Create contact and assign 3 companies to this contact
2. Create form with action with add points. Send two times these form for same contact. Then setup contact point log report
3. Setup data table with email and company name
![image](https://user-images.githubusercontent.com/462477/44510769-d81ef900-a6b5-11e8-88af-7060698ad4fd.png)
4. Setup graphs table: Contacts with most points
![image](https://user-images.githubusercontent.com/462477/44510789-e79e4200-a6b5-11e8-943d-97fdaa7addfd.png)
5. See graphs table display more points like  your contact submitted via form

#### Steps to test this PR:
1.  Apply PR 
2. See correct points on graphs table

![image](https://user-images.githubusercontent.com/462477/44511043-8f1b7480-a6b6-11e8-95c5-ad8224c6beec.png)
